scripts/gulp/lint.js

```diff
@@ -33,17 +34,28 @@ gulp.task( 'lint:build', function() {
  * Lints the test js files for errors.
  */
 gulp.task( 'lint:tests', function() {
-  return _genericLint( [
+  return _genericJSLint( [
     'test/accessibility/*.js',
     'test/*.js'
   ] );
 } );
 
 /**
- * Lints the source js files for errors.
+ * Lints the source LESS files for errors.
  */
 gulp.task( 'lint:scripts', function() {
-  return _genericLint( [ 'src/**/src/*.js' ] );
+  return _genericJSLint( [ 'src/**/src/*.js' ] );
+} );
+
+gulp.task('lint:less', function() {
+ 
+  return gulp
+    .src( ['!src/cf-grid/src-generated/*.less', 'src/**/*.less'] )
```

Comment:
Why specifically cf-grid?

Reply:
Is this a generated file? I ignored it thinking that it was. If not, I will re-add it.

Reply:
Ohhh, duh, sorry, I misinterpreted this.